cmd/starsd/cmd/init.go

```diff
@@ -150,7 +152,14 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 				return fmt.Errorf("genesis.json file already exists: %v", genFile)
 			}
 
-			appState, err := json.MarshalIndent(mbm.DefaultGenesis(cdc), "", " ")
+			gen := mbm.DefaultGenesis(cdc)
+			sgDefaultGen := sg.NewDefaultGenesisState(cdc)
+			gen["tokenfactory"] = sgDefaultGen["tokenfactory"]
+			// for key := range sgDefaultGen {
```

Comment:
Is this commented code here for a reason?

Reply:
By default the token factory genesis data encodes a denom creation fee of `1stake`, which we don't want. I couldn't for the life of me figure out how to override this genesis value without these ugly changes. 

Happy to pair with someone to figure it out.